packs/data/spells.db/miracle.json

```diff
@@ -26,7 +27,7 @@
             "value": {}
         },
         "description": {
-            "value": "<p>You request aid directly from your divine source. Your divine source always refuses a request out of line with its nature, and it might grant a different request (potentially more powerful or better fitting its nature) than the one you asked for. A casting of <em>miracle</em> can do any of the following things.</p>\n<ul>\n<li>Duplicate any divine spell of 9th level or lower.</li>\n<li>Duplicate any non-divine spell of 7th level or lower.</li>\n<li>Produce any effect whose power level is in line with the above effects.</li>\n<li>Reverse certain effects that refer to the <em>wish</em> spell.</li>\n</ul>\n<p>The GM might allow you to try using <em>miracle</em> to produce greater effects than these, but doing so may be dangerous, or the spell may have only a partial effect.</p>"
+            "value": "<p>You request aid directly from your divine source. Your divine source always refuses a request out of line with its nature, and it might grant a different request (potentially more powerful or better fitting its nature) than the one you asked for. A casting of <em>miracle</em> can do any of the following things.</p>\n<ul>\n<li>Duplicate any divine spell of 9th level or lower to which you have access.</li>\n<li>Duplicate any non-divine spell of 7th level or lower to which you have access.</li>\n<li>Produce any effect whose power level is in line with the above effects.</li>\n<li>Reverse certain effects that refer to the @Compendium[pf2e.spells-srd.Wish]{Wish} spell.</li>\n</ul>\n<p>The GM might allow you to try using <em>miracle</em> to produce greater effects than these, but doing so may be dangerous, or the spell may have only a partial effect.</p>"
```

Comment:
```suggestion
            "value": "<p>You request aid directly from your divine source. Your divine source always refuses a request out of line with its nature, and it might grant a different request (potentially more powerful or better fitting its nature) than the one you asked for. A casting of <em>miracle</em> can do any of the following things.</p>\n<ul>\n<li>Duplicate any divine spell of 9th level or lower to which you have access.</li>\n<li>Duplicate any non-divine spell of 7th level or lower to which you have access.</li>\n<li>Produce any effect whose power level is in line with the above effects.</li>\n<li>Reverse certain effects that refer to the <em>@Compendium[pf2e.spells-srd.Wish]{Wish}</em> spell.</li>\n</ul>\n<p>The GM might allow you to try using <em>miracle</em> to produce greater effects than these, but doing so may be dangerous, or the spell may have only a partial effect.</p>"
```